apps/coscrad-frontend/src/coscrad-theme-provider.tsx

```diff
@@ -8,13 +7,16 @@ interface CoscradThemeProviderProps {
 }
 
 export const CoscradThemeProvider = ({ children }: CoscradThemeProviderProps): JSX.Element => {
-    const _configurableContent = useContext(ConfigurableContentContext);
+    const { theme } = useContext(ConfigurableContentContext);
 
     const coscradDefaultTheme = responsiveFontSizes(
         createTheme({
             palette: {
                 primary: {
-                    main: blue[800],
+                    main: theme.primaryColor,
```

Comment:
I don't like this mapping from your "theme" property to the actual theme properties. It will be hard to maintain (not extensible) and error prone.

If you are just going to flow through, I suggest that you
- name the config property `themeOverrides`
- define a type for this `ConfigurableThemeOverrides` which is build from picking \ exluding or just a partial of the MUI Theme type
then just "spread in" the overrides here.

Reply:
Now fixed